recipes/pydtmc/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "PyDTMC" %}
+{% set version = "6.2.0" %}
+{% set user = "TommasoBelluzzo" %}
```

Comment:
Don't templatize `user` since it doesn't really help make the recipe flexible and it will just complicate things when we need to add maintainers in the future.